lib/forwarder.js

```diff
@@ -12,15 +12,22 @@ import { removeConfig } from '../lib/config.js';
  * @param {Config} config
  */
 export async function forwardToDaemon(resolver, config) {
+  let eslintArgs = [...process.argv];
   const text = process.argv.includes('--stdin') ? await readStdin() : null;
+  const fixToStdout = Boolean(process.argv.includes('--fix-to-stdout'));
   const { stdout } = supportsColor;
 
+  if(fixToStdout) {
+    eslintArgs = [eslintArgs[0], eslintArgs[1], "--fix-dry-run", "--format", "json", ...eslintArgs.slice(2)];
+    eslintArgs = eslintArgs.filter((arg) => arg !== "--fix-to-stdout");
```

Comment:
How about using `indexOf('--fix-to-stdout)`, check if the index is not -1, and then use `splice(index, 1, '--fix-dry-run', '--format', 'json')`?

Reply:
done, thank you